src/seedsigner/views/tools_views.py

```diff
@@ -241,6 +250,189 @@ def run(self):
 
 
 
+"""****************************************************************************
+    Coin flips Views
+****************************************************************************"""
+class ToolsCoinEntropyMnemonicLengthView(View):
+    def run(self):
+        # TRANSLATOR_NOTE: Inserts the number of coin flips needed for a 12-word mnemonic
+        twelve = _("12 words ({} flips)").format(mnemonic_generation.COIN__NUM_FLIPS__12WORD)
+        TWELVE = ButtonOption(twelve, return_data=mnemonic_generation.COIN__NUM_FLIPS__12WORD)
+
+        # TRANSLATOR_NOTE: Inserts the number of coin flips needed for a 24-word mnemonic
+        twenty_four = _("24 words ({} flips)").format(mnemonic_generation.COIN__NUM_FLIPS__24WORD)
+        TWENTY_FOUR = ButtonOption(twenty_four, return_data=mnemonic_generation.COIN__NUM_FLIPS__24WORD)
+
+        button_data = [TWELVE, TWENTY_FOUR]
+        selected_menu_num = ButtonListScreen(
+            title=_("Mnemonic Length"),
+            is_bottom_list=True,
+            is_button_text_centered=True,
+            button_data=button_data,
+        ).display()
```

Comment:
Ah, here's the problem. You had the bad luck of basing this on older `View` code that was not yet updated for the `self.run_screen()` call that is required for the flow tests. If a View still use the old `Screen.display()` method, that means that a flow test has not yet been written for that View, which indicates a testing gap. ALL Views should be updated to the `self.run_screen()` method.

```suggestion
        selected_menu_num = self.run_screen(
            ButtonListScreen,
            title=_("Mnemonic Length"),
            is_bottom_list=True,
            is_button_text_centered=True,
            button_data=button_data,
        )
```